shell/components/LabelsPSA.vue

```diff
@@ -0,0 +1,72 @@
+<script>
+import KeyValue from '@shell/components/form/KeyValue';
+import { ToggleSwitch } from '@components/Form/ToggleSwitch';
+import { getPSALabels } from '@shell/utils/pod-security-admission';
+
+export default {
+  components: {
+    ToggleSwitch,
+    KeyValue
+  },
+  props: {
+    value: {
+      type:     Object,
+      required: true,
+    },
+    mode: {
+      type:     String,
+      required: true,
+    },
+  },
+  data() {
+    return { toggler: true };
+  },
+  computed: {
+    /**
+     * Generate list of present keys which can be filtered based on existing label keys and system keys
+     */
+    protectedKeys() {
+      return getPSALabels(this.value);
```

Comment:
This one can be moved to the model for the value type as well

Reply:
How can I access to the PSA here? This resource (`value`) is a Namespace in this case, so it has not the method. Or are you asking me to write it twice for the Namespace and the PSA?

Reply:
The getter method can be added to the namespace model. This is the only place where `getPSALabels` is used

Reply:
Moved within model.

Reply:
Resolved by https://github.com/cnotv/dashboard/pull/7